Directory.Build.props

```
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup Label="Build">
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6</TargetFramework>
```

The target framework is `net6.0`

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```